1/backend/src/resolvers.js

```diff
@@ -1,67 +1,268 @@
     },
     Mutation: {
-        addTodo: (parent, args, context) => {
-            if(args.message != "" && args.message != null && context.token) {
-                let newTodo = {
-                    id: findNextId(),
+        addTodo: async (parent, args, context) => {
+            if (args.message != "" && args.message != null && context.token) {
+                const {driver} = context
+                const createTodoCypher = `
+                    CREATE (todo:Todo {params})
+                    RETURN todo.id, todo.message, todo.completed
+                `
+                const params = {
                     message: args.message,
                     completed: false,
-                };
-                data.todos.push(newTodo);
-                return newTodo;
+                    id: uuid()
+                }
+                const session = driver.session()
+                try {
+                    data = await session.run(createTodoCypher, {params})
+                    const [todo] = await data.records.map(record => ({
+                        id: record.get('todo.id'),
+                        message: record.get('todo.message'),
+                        completed: record.get('todo.completed')
+
+                    }))
+                    return todo;
+                } finally {
+                    await session.close()
+                }
             }
             return;
         },
-        deleteTodo: (parent, args, context) => {
-            if (args.id != null && context.token){
-                let index = data.todos.findIndex(td => td.id === parseInt(args.id));
-                if (index > 0){
-                    data.todos.splice( index, 1)
+        addUser: async (parent, args, context) => {
+            if (args.login != "" && args.login != null && args.password != "" && args.password != null) {
+                const {driver} = context
+                const createUserCypher = `
+                    CREATE (user:User {params})
+                    RETURN user
+                `
+                const session = driver.session()
+                try {
+                    await session.run(createUserCypher)
+                } finally {
+                    await session.close()
+                }
+                return true
+            }
+            return false;
+        },
+        deleteUser: async (parent, args, context) => {
+            if (args.login != "" && args.login != null && args.password != "" && args.password != null) {
+                const {driver} = context
+                const createUserCypher = `
+                    MATCH (user:User {params})
+                    DELETE user
+                `
+                const session = driver.session()
+                try {
+                    await session.run(createUserCypher)
+                } finally {
+                    await session.close()
+                }
+                return true
+            }
+            return false;
```

Comment:
You can return `null` if the user could not be found. Return the deleted user if it was a success.